### PR TITLE
Homepage/sidebar

### LINKED ISSUE #3: Storybook: Homepage Account Sidebar Menu component
## User story

**Who:** **As a** user
**What:** **I want to** be able to use the options on the side menu 
**Why:** **so that I can** I can choose between registering a new account or login into your account

## Acceptance criteria

Create a storybook component sidebar like the one we can see highlighted in pink 
 
Screenshot 2021-04-12 at 22 37 58

Check the wireframes here: https://www.figma.com/file/9SJRaxPSuPI3lSl8ULNyJ8/GuideIT-For-Developers?node-id=0%3A1

## Implementation details

Use the font [Chakra Petch](https://fonts.google.com/specimen/Chakra+Petch?preview.text=GuideIT&preview.text_type=custom) 
The background color is #0B5681
Check the example here in how to create triangles https://www.w3schools.com/howto/tryit.asp?filename=tryhow_css_shapes_triangle-right
Use the logos below, HYF is white so you might not see it, and add the files to the project on this folder /src/client/assets/images/

HYF-logo
redi-logo

### PR DESCRIPTION
**Pull request by DlnyaMazhari**
# Description

this PR solves issue#3 for homepage sidebar component.

Fixes #3 

# How to test?

npm run storybook 

# Checklist

- [x ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/SidebarMenu/SidebarMenu.stories.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import SidebarMenu from './SidebarMenu';
+
+export default {
+  title: 'SidebarMenu',
+  component: 'SidebarMenu',
```
Comment (Enqira) [inline src/client/components/SidebarMenu/SidebarMenu.stories.js:6]: 'SidebarMenu' in component should be without single quotes.

**Review by Enqira — CHANGES_REQUESTED** [review]
Great job, besides that small detail about component name everything looks good

File: src/client/components/SidebarMenu/SidebarMenu.stories.js (hunk 1 above)
Comment (DlnyaMazhari) [reply src/client/components/SidebarMenu/SidebarMenu.stories.js]: thanks , it is changed.

**Review by Enqira — APPROVED** [review]

**Review by dpfernandes — CHANGES_REQUESTED** [review]
@DlnyaMazhari please check PR #68 and once it's merged use those fonts instead.

File: src/client/components/SidebarMenu/SidebarMenu.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+.triangle {
+  width: 0;
+  height: 0;
+  border-top: 8px solid transparent;
+  border-left: 16px solid rgb(230, 97, 20);
+  border-bottom: 8px solid transparent;
+  margin-left: 15px;
+}
```
Comment (TITHI1244) [inline src/client/components/SidebarMenu/SidebarMenu.css:48]: I think, we do not need to create the triangle. You can find it as an image and download it from figma :)

File: src/client/components/SidebarMenu/SidebarMenu.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
+@import url('https://fonts.googleapis.com/css2?family=Chakra+Petch:wght@300;600&display=swap');
```
Comment (TITHI1244) [inline src/client/components/SidebarMenu/SidebarMenu.css:1]: Remember to delete the import as Daniel asked :)

**Review by TITHI1244 — CHANGES_REQUESTED** [review]
Nicely done @DlnyaMazhari. Very minor changes to be done and ready to merge :)

**Review by TITHI1244 — APPROVED** [review]

**Review by dpfernandes — APPROVED** [review]
👍

**Review by soujanyapolavarapu — APPROVED** [review]
Good job @DlnyaMazhari

**Comment by DlnyaMazhari** [comment]
> Good job @DlnyaMazhari

thanks Soujanya

**Pull request merged**